### PR TITLE
perf(*) reduce ngx.var usage

### PR DESCRIPTION
**Pull request by fffonion**
### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: kong/runloop/handler.lua [hunk 1]
```diff
@@ -1113,11 +1113,6 @@ return {
     before = function(ctx)
       local server_port = var.server_port
       ctx.host_port = HOST_PORTS[server_port] or server_port
-
```
Comment (fffonion) [inline kong/runloop/handler.lua:1116]: removed as it seems not being read afterwards

Comment (bungle) [reply kong/runloop/handler.lua:1116]: that was because of a bug

**Comment by guanlan** [comment]
@fffonion is this still relevant?

**Comment by fffonion** [comment]
Closing as a different approach has replaced this.

**Pull request closed without merging**